### DISCUSSION TITLE
Password prompt when using public key authentication in SSH tunnel

### DISCUSSION THREAD

**Discussion by alexei-znamensky-movile** [dbeaver/dbeaver#179] [Q&A] [answered]
Even though I have a private key configuration in place, it still asks me for a password upon connection. I click on "OK" and the connection happens, but nonetheless it is simply an annoyance having to do that _everytime_ I connect.

**Comment by serge-rider** [comment, marked as answer]
To disable password prompt just leave password empty and enable "Save password" checkbox.

This field (password) is visible for private-key based authentication because private key can also be password protected.
